components/playlist/browser/resources/components/playerSeeker.tsx

```diff
@@ -0,0 +1,253 @@
+// Copyright (c) 2023 The Brave Authors. All rights reserved.
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this file,
+// You can obtain one at https://mozilla.org/MPL/2.0/.
+
+import * as React from 'react'
+
+import styled from 'styled-components'
+import { color, font } from '@brave/leo/tokens/css'
+
+interface Props {
+  videoElementRef: React.RefObject<HTMLVideoElement>
+}
+
+const SeekerContainer = styled.div`
+  display: flex;
+  flex-direction: column;
+`
+
+const TimeContainer = styled.div`
+  display: grid;
+  grid-template-columns: auto auto;
+  justify-content: space-between;
+
+  color: ${color.text.secondary};
+  font: ${font.primary.xSmall.regular};
+`
+
+const StyledProgress = styled.progress`
+  --progress-background: #423eee; /* TODO(sko) Neither color.button.background or color.semantic.button.background exist for now */
```

Comment:
Yeah, Agustin is doing a huge token refactor to get them to work with the built in style tokens in Figma :fearful: Hopefully we can get it sorted soon